docs/pages/raster_datasets.rst

```diff
@@ -0,0 +1,56 @@
+Working with Raster Datasets
+====================================
+
+.. Note:: Before using this quick guide, it will help your understanding to do the :doc:`Basic Usage Tutorial </pages/basic_usage_tutorial>`.
+
+Dataset type
+~~~~~~~~~~~~
+Raster datasets are currently implemented as :doc:`Raster Datasets V1 </pages/development/raster_v1>`. Kart version controls raster tiles themselves as GeoTIFF files, which means the tiles are checked out into the working copy without any conversion step - the contents of the working copy is an accurate reflection of the contents of the dataset itself at a particular point in time.
```

Comment:
```suggestion
Kart internally stores raster tiles as GeoTIFF files, which means the tiles are checked out into the working copy without any conversion step - the contents of the working copy is an accurate reflection of the contents of the dataset itself at a particular point in time.
 
To find out more about the internal storage of raster datasets, see :doc:`Raster Datasets V1 </pages/development/raster_v1>`.
```

I mostly want to de-emphasise the linked doc for beginning users - it's probably not relevant to them / might be confusing